client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.jsx

```diff
@@ -15,13 +15,15 @@ export const ToolbarButton = ({
   className,
 }) => {
   const iconClassName = `icon-action icon-${iconName}`;
-  const buttonClassName = className
-    ? `data-files-toolbar-button ${className}`
-    : 'data-files-toolbar-button';
   return (
-    <Button disabled={disabled} onClick={onClick} className={buttonClassName}>
-      <i className={iconClassName} data-testid="toolbar-icon" />
-      <span className="toolbar-button-text">{text}</span>
+    <Button
+      iconNameBefore={iconClassName}
+      size={['Compress', 'Download'].includes(text) ? 'long' : 'medium'}
+      type={text === 'Empty' ? 'primary' : 'secondary'}
```

Comment:
👍 Well done.

[I ask Design privately about these buttons' widths.](https://tacc-team.slack.com/archives/G01K9ET7JM7/p1657838165752589) Even with your _size optimization logic_, the toolbar is overall wider than before.

<details>

**Before**
![Medium Screen (1200px) - Before](https://user-images.githubusercontent.com/62723358/179100259-557058e1-8cd7-4054-a252-067d009b14a3.png)

**After**
![Medium Screen (1200px) - After](https://user-images.githubusercontent.com/62723358/179100265-41acea56-28ce-449d-b032-6807bba3a417.png)

</details>